qiskit/visualization/pulse/matplotlib.py

```diff
@@ -31,8 +31,8 @@
 from qiskit.pulse.channels import (DriveChannel, ControlChannel,
                                    MeasureChannel, AcquireChannel,
                                    SnapshotChannel)
-from qiskit.pulse import (SamplePulse, FrameChange, PersistentValue, Snapshot,
-                          Acquire, PulseError)
+from qiskit.pulse import (SamplePulse, FrameChange, SetFrequency, PersistentValue,
```

Comment:
If it's not too much trouble, could you share an example of how `SetFrequency` is drawn?